bin/FvwmPrompt/meson.build

```diff
@@ -0,0 +1,33 @@
+# Fvwmprompt is written in golang. Unfortunately there is still
+# no automagic in meson to handle golang projects. So we have to
+# do it manually. It'll be fine™
+
+# Go Go Gadget Golang hacks
+go_env = environment()
+# TODO: These LDFLAGS were pulled straight from the legacy Makefile
+#       I'm not sure if they're necessary or not.
+go_env.set('CGO_LDFLAGS', '-s -w')
+go_env.set('CGO_CFLAGS',  '')
+go_env.set('CGO_CFLAGS_ALLOW', '.*')
+go_env.set('CGO_LDFLAGS_ALLOW', '.*')
+
+# Extract all of the go sources from the current directory
+r = run_command('find-go-sources.sh', meson.current_source_dir(), check: true)
+fvwmprompt_files = r.stdout().strip().split()
```

Comment:
Is there no better way to do this other than with an external script? This seems very unnecessary and clunky.

Reply:
As far as I am aware, upstream golang has never implemented "depfile" support and therefore this is indeed needed.